docs/guides/admin/docs/installation/ansible.md

```diff
@@ -0,0 +1,45 @@
+Install from Repository via Ansible (Debian, and RedHat based distros)
+===========================================================================
+
+We provide [Ansible](https://www.ansible.com/) installation and configuration scripts which drastically reduce the time
+required to set up an Opencast installation, and help you manage the configuration across your entire cluster.  Whether
+you are installing a single machine, or dozens, these scripts should do the basic setup of your Opencast install.  These
+scripts use the package repository, so the first step is to register.
+
+Note: Examples in this document refer to `7.x` and `r/7.x`.  You should replace those with the version you want to install!
+
+Registration
+------------
+
+Before you can start you need to get an account for the repository. You will need the credentials that you get by mail
+after the registration to successfully complete this manual. The placeholders `[your_username]` and `[your_password]`
+are used in this manual wherever the credentials are needed.
+
+[Please visit https://pkg.opencast.org](https://pkg.opencast.org)
+
+
+Install Ansible
+---------------
+
+Ansible may be available from your distribution's packaging manager, or you may need to install it manually.  We
+recommend following the [Ansible installation documentation](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html)
+to get the latest version.
+
+
+Script Setup
+------------
+
+The next step is getting the scripts themselves.  Go to the [list of branches](https://github.com/opencast/oc-config-management/branches)
```

Comment:
I'm not entirely happy with this section but it should do and I don't really mind keeping it.

To use an installation method like this, you should really know what you are doing. More than when you do it manually. No one should just run these scripts. This means that we can expect people to have a certain technical skill level and it would just suffice to point them to the GitHub repo without explaining how to click a download button ;-)

Reply:
I wasn't really sure how to do this part.  We need a better technical writer, but this will do for now.